src/ResourceManager/Websites/AzureRM.Websites.Netcore.psd1

```diff
@@ -93,8 +93,7 @@ CmdletsToExport = 'Get-AzureRmAppServicePlan', 'Set-AzureRmAppServicePlan',
                'New-AzureRmWebAppBackup', 'Remove-AzureRmWebApp', 
                'New-AzureRmWebApp', 'Remove-AzureRmWebAppBackup', 
                'Reset-AzureRmWebAppPublishingProfile', 'Restart-AzureRmWebApp', 
-               'Set-AzureRmWebApp', 'Start-AzureRmWebApp', 'Stop-AzureRmWebApp',
-               'Get-AzureRmWebAppSnapshot', 'Restore-AzureRmWebAppSnapshot'
```

Comment:
Does removing the commands from here remove this from the help as well or do we need to remove the .md files as well so that the MSDN documentation doesn't pick up these commands